client/coral-embed-stream/src/tabs/stream/containers/DraftArea.js

```diff
@@ -71,4 +77,16 @@ DraftAreaContainer.propTypes = {
   disabled: PropTypes.bool,
   rows: PropTypes.number,
   label: PropTypes.string.isRequired,
+  comment: PropTypes.object,
 };
+
+const slots = ['commentBox'];
```

Comment:
I think we should rename this slot to `draftArea` because it confuses with our `CommentBox` container which is only used for `post` and `replies` but not for `edit`